proto/spfresh.proto

```
@@ -42,11 +50,75 @@ message SPFreshSearchRequest {
+}
+
+message SPFreshFilterExprColumn {
+    int64 column_id = 1;
+    SPFreshFilterExprColumnSource source = 2;
+    FieldType field_type = 3;
 }
 
 message SPFreshSearchResponse {
+    oneof result {
+        SPFreshSearchResult success = 1;
+        Error error = 2;
```

Buf (1.72.0)

[error] cannot find `Error` in this scope

(COMPILE)